physics/module_sf_noahmp_glacier.F90

```diff
@@ -2652,7 +2652,7 @@ subroutine snowwater_glacier (nsnow   ,nsoil   ,imelt    ,dt     ,sfctmp , & !in
 
 !to obtain equilibrium state of snow in glacier region
        
-   if(sneqv > mwd) then   ! 100 mm -> maximum water depth
+   if(sneqv > mwd .and. isnow /= 0) then   ! 100 mm -> maximum water depth
```

Comment:
@barlage @HelinWei-NOAA Are you both OK with adding this to avoid a floating point exception that is sometimes triggered (e.g. in the rrfs_v1beta_debug_intel test for this PR and in previous PRs: https://github.com/ufs-community/ccpp-physics/issues/114)?

Reply:
@grantfirl yes, I'm good with this change. I was going to suggest bumping the mwd up to a larger number again, but this should fix the issue. It's still puzzling to me as to why this happens. Will look into it more to track down the root cause.